meson.build

```diff
@@ -237,12 +237,14 @@ else
   message('Using bundled openssl code')
 endif
 
+has_debugger = not (host_machine.system() == 'linux' and host_machine.cpu_family() == 's390x')
```

Comment:
can we move this into a separate function, there will be more cases that we dont support the debugger. But in theory, r2 should build in those platforms, but the debug plugin should do nothing and warn at runtime.

Reply:
https://mesonbuild.com/Syntax.html#userdefined-functions-and-methods it's not possible, sorry. This is why I have defined this variable. I agree that there should be an error at runtime though. We should have an "empty" debug plugin or something.